Dashboard Mockup

rasta-dave opened the pull request:
Integrated dashboard with Vite. Updated the dashboard prototype to display mock data and ready for starting API integration. Updated dashboard metrics to better reflect Braidpool's consensus mechanism. Restructured code into logical directories for better maintainability. Created a template that is modular and can be improved upon UX appearance wise later on

![Screenshot From 2025-04-05 15-58-39](https://github.com/user-attachments/assets/39795aff-0d4b-4d9c-905c-46cf51f27687)
![Screenshot From 2025-04-05 15-59-07](https://github.com/user-attachments/assets/0cd2bdec-a398-40e7-aba2-2c27a3bee7e7)
![Screenshot From 2025-04-05 15-59-27](https://github.com/user-attachments/assets/bdf69a4f-a536-4356-8c64-fdd313c2a2ff)
![Screenshot From 2025-04-05 15-59-33](https://github.com/user-attachments/assets/b47cb77c-2b7e-474b-956c-7924944debc6)
![Screenshot From 2025-04-05 15-59-37](https://github.com/user-attachments/assets/4f21ba8f-b8df-4839-9bbc-edde9aba3791)
![Screenshot From 2025-04-05 15-59-43](https://github.com/user-attachments/assets/e9f2221c-e7b6-4320-9b6f-64a03f0caac7)

its-me-abhishek commented on dashboard/robots.txt:
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
this file should be removed imo, if you were using some assistant

priyashuu reviewed (changes requested):
Handle missing data – Show a placeholder if hashrateData is empty.

priyashuu reviewed (commented):
Handle missing data – Show a placeholder if hashrateData is empty.

priyashuu commented:
@rasta-dave You should raise the PR against the dev branch. I know you already know that, but just a reminder. Otherwise, the UI looks good! 👀🚀

rasta-dave commented:
I've addressed all the feedback from @its-me-abhishek and @priyashuu - removed the robots.txt file and reorganized the components structure as suggested. I've also updated the base branch to dev as requested.

its-me-abhishek reviewed (approved):
LGTM! I think that this is a good enough starting point. I will work on the directory structure and remove the test_braids after integrating the api based graph component into this
